simulation/g4main/PHG4ParticleGun.cc

```diff
@@ -52,16 +51,18 @@ PHG4ParticleGun::process_event(PHCompositeNode *topNode)
 	if(_beam_profile) {
 		_beam_profile->GetRandom2(vx, vy);
 	}
-	
-	
-
-
+		
   PHG4InEvent *ineve = findNode::getClass<PHG4InEvent>(topNode,"PHG4INEVENT");
   ReuseExistingVertex(topNode); // checks if we should reuse existing vertex
 
-
-  // cout<<" Here I am in particle gun...."<<endl;
-  //cout << "PHG4ParticleGun: {" << vx << ", " << vy << ", "<<vtx_z<< "}" << endl;
+  //! using vertex generated by E906 legacy generator
+  if ( _legacy_vertexgenerator) 
+	{TVector3 vtx        = _vertexGen->generateVertex();
```

Comment:
Better make a new line after `{`.

Reply:
I will change it accordingly. Thank you.